frontend/viewer/src/ProjectView.svelte

```diff
@@ -217,6 +217,8 @@
     $entries?.push(entry);//need to add it before refresh, otherwise it won't get selected because it's not in the list
     if (!options?.dontNavigate) {
```

Comment:
now that there's an else can we flip this condition? `if not don't navigate` is a double negative. If we flip this then it can just read `if don't navigate`

Reply:
I think we could probably also just merge this, because we're going to revisit everything and I've suggested always refreshing (i.e. removing the `else`) in #1372:

> In the 2 cases "on new entry created by us" and "on new entry from a different user" we have the problem of knowing if the entry should be in the current filtered list of entries on our machine. I can think of 2 fixes:
> 1) Ultra-complex pro fix: we ask the server if entry X matches filter Y and what index it has in the matching result list, so we can insert it ourselves. Yikes.
> 2) Quick fix: we always just refresh the list of entries and it'll show up if it should. 👈 I'd be inclined to implement this as part of this issue.

Reply:
Agree on merging this, and I'll add a note to #1372 mentioning wanting to flip the check (any time you see `if (!dontFoo)`, you should ask yourself whether you can make it `if (foo)` instead).